.circleci/config.yml

```diff
@@ -19,6 +19,22 @@ jobs:
       - run: yarn test:lint
       - run: yarn test:prettier
 
+  publish_main_tag:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "yarn.lock" }}
+      - run:
+          name: Build
+          command: yarn build
+      - run:
+          name: Set version
+          command: yarn version --no-git-tag-version --new-version 0.0.0-main-$(echo $CIRCLE_SHA1 | cut -c -7)
+      - run: npm publish --tag main
```

Comment:
Did you set up circleci with permission to publish to the repo?

Reply:
I had an impression it works because `--dry-run` wasn't reporting anything but probably it would fail when publishing.

4249e41 

I added the token to circleci.